src/passes/Print.cpp

```diff
@@ -454,7 +454,7 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     if (curr->params.size() > 0) {
       for (size_t i = 0; i < curr->params.size(); i++) {
```

Comment:
instead of calling toIndex, we can just use Index as the type of i. I suppose then we'd need toIndex on the size() output? that is still less code clutter though.

but I would like to not even have the toIndex on the size() call. The array type could be modified, perhaps?